packages/vxrn/src/create.ts

```diff
@@ -49,7 +50,9 @@ const extensions = [
 ]
 
 export const create = async (options: StartOptions) => {
-  const { host = '127.0.0.1', root, nativePort = 8081, webPort } = options
+  const { host = '127.0.0.1', root, port = 8081 } = options
+  const nativePort = await getPort()
```

Comment:
I don't see it being used anywhere, so maybe we can remove it?

Reply:
Yea lots of old test code feel free to keep going